content/docs/reference-dom-elements.md

```diff
@@ -73,70 +73,70 @@ const divStyle = {
 };
 
 function HelloWorldComponent() {
-  return <div style={divStyle}>Hello World!</div>;
+  return <div style={divStyle}>Witaj, świecie!</div>;
 }
 ```
 
-Note that styles are not autoprefixed. To support older browsers, you need to supply corresponding style properties:
+Zauważ, że style nie mają automatycznie nadawanego prefiksu. Aby zapewnić wparcie dla starszych przeglądarek, musisz nadać odpowiednie właściwości:
```

Comment:
```suggestion
Zauważ, że style nie mają automatycznie nadawanego prefiksu. Aby zapewnić kompatybilność starszych przeglądarek, musisz nadać im odpowiednie właściwości:
```

Reply:
W takim razie powinno być "Aby zapewnić kompatybilność ze starymi przeglądarkami,...". To React dopasowuje się do starych przeglądarek, a nie przeglądarki do Reacta :)